reference/glossary.ditamap

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
+<map>
+    <title>Glossary entries</title>
+    <topicref href="glossary.dita" format="dita" scope="local" type="concept" keys="glossary" chunk="to-content">
+        <glossref href="gloss-argument.dita" format="dita" scope="local" type="glossentry" keys="gloss-argument"
+            print="yes"/>
+        <glossref href="gloss-dost.dita" format="dita" scope="local" type="glossentry" keys="gloss-dost" print="yes"/>
+        <glossref href="gloss-dita-open-toolkit.dita" format="dita" scope="local" type="glossentry"
+            keys="gloss-dita-open-toolkit" print="yes"/>
+        <glossref href="gloss-extension-point.dita" format="dita" scope="local" type="glossentry"
+            keys="glosss-extension-point" print="yes"/>
```

Comment:
To simplify maintenance and remain consistent with the other map files in the project, it would be better to add each file to `source-files.ditamap`, define the keys there and pull them in to `glossary.ditamap` via `@keyref` instead of `@href`.

The redundant `@format` & `@scope` attributes can also be dropped here.